JavaScript/JavaScript.sublime-syntax

```diff
@@ -136,6 +133,26 @@ contexts:
         - match: \n
           pop: true
 
+  jsdoc-comment:
+    - meta_include_prototype: false
+    - meta_scope: comment.block.documentation.js
+    - match: \*/
+      scope: punctuation.definition.comment.end.js
+      pop: 1
+    # JSDoc "block" tags (i.e. @param) are only accepted at the beginning of the documentation
+    # line so directly after the '/**' or after the '*' marker on the next lines.
+    - match: ^\s*(\*)(?!/)
+      captures:
+        1: punctuation.definition.comment.js
+      push: jsdoc-block-tag
+
+  jsdoc-block-tag:
+    - match: '{{jsdoc_block_tag}}'
+      scope: meta.tag.jsdoc entity.other.attribute-name.documentation.jsdoc
```

Comment:
I would probably drop the `meta` scope unless you are going to scope the entire line with it. The suffix on the scope should just be `js`, IMO.

```suggestion
      scope: entity.other.attribute-name.documentation.js
```